src/lib.rs

```diff
@@ -73,6 +77,12 @@ where
     }
 }
 
+impl ToMetricInst<i32> for HostStat<HealthCheckStat> {
+    fn to_metric_inst(&self) -> PrometheusInstance<'_, i32, Yes> {
+        PrometheusInstance::new().with_value(self.value.healthy as i32)
```

Comment:
It would be nice to provide the unhealthy targets (`targets` field of `HealthCheckStat`).

We can exports multiple time the metrics with different labels, something like:
```
# HELP lustre_health_check_stats Gives information about Lustre health check status.
# TYPE lustre_health_check_stats gauge
lustre_health_check_stats{target="test-OST0000"} 0
lustre_health_check_stats{target="test-OST0001"} 0
```
Lustre does not report healthy target, so we will never have:
```
lustre_health_check_stats{target="test-OST0000"} 1
```